packages/asc-ui/src/components/Link/Link.tsx

```diff
@@ -1,11 +1,27 @@
-import { styledComponents } from '@datapunt/asc-core'
 import React from 'react'
-import LinkStyle from './LinkStyle'
+import { styledComponents } from '@datapunt/asc-core'
+import { ChevronRight } from '@datapunt/asc-assets'
+import LinkStyle, { Props as LinkStyleProps } from './LinkStyle'
+import Icon from '../Icon'
+
+export type Props = {
+  noIcon?: boolean
+} & styledComponents.StyledProps<any> &
+  LinkStyleProps
 
-export type Props = styledComponents.StyledProps<any>
+const Link: React.FC<Props> = ({
+  children,
+  noIcon,
+  linkType,
+  ...otherProps
+}) => (
+  <LinkStyle $as="a" {...otherProps} linkType={linkType}>
+    {linkType === 'default' && !noIcon && (
```

Comment:
Why specify an icon with the default type instead of having an extra type like `'default' | 'inline' | 'blank' | 'default-chevron'`? As the chevrons will be removed from the default link in the design system

Reply:
Good point! I have also thought about this option myself. I'll implement it this way.